docs/kube/secret.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: brokersecret
+type: Opaque
+data:
+  username: YnJva2Vy
+  password: cGFzc3dvcmQ=
```

Comment:
Is this username and password hardcoded to this always? Or is this just a sample one for your deployment?

Reply:
its assuming you're using the default username/password...like it says in the README.

Reply:
username appears to be `base64` too. should be explicit in the doc.

Reply:
never mind. comes in the next sentence.

Reply:
`echo broker | base64` gives me `YnJva2VyCg==`

Reply:
fixed